Applications/VideoPlayer/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+set(SOURCES
+	main.cpp
+    VideoPlayer.cpp
+    FrameWidget.cpp
```

Comment:
Some issues with formatting here. Check tabs/spaces and use the same as in other CMake files ?